galley/pkg/config/analysis/metadata.go

```diff
@@ -20,6 +20,9 @@ import (
 
 // Metadata represents metadata for an analyzer
 type Metadata struct {
-	Name   string
-	Inputs collection.Names
+	Name string
+	// Description explains what the analyzer checks in a single sentence (shown
+	// when someone runs `istioctl x analyze -L`).
+	Description string
```

Comment:
Leave out the mention of `istioctl x analyze` since it may not always be relevant or accurate. (At the very least, we'll eventually drop the `x`. :) ) Documenting best practices for what the description should be should go in the analyzer README anyway, I think.

Reply:
Done.